scripts/install-lib.sh

```diff
@@ -1,40 +1,21 @@
 #!/bin/bash
 
-OS=$(uname -s)
-if [ "$OS" = "Linux" ]; then
-  # see if package already installed
-  V=$(dpkg -l libatlasclient | awk '/^ii/{print $3}')
-  case $V in
-    2.2.* )
-      # assume the rest of the dependencies have been installed
-      echo libatlasclient already installed: $V
-      ;;
-    * )
-      # same package name, but the xenial ami doesn't ship g++
-      ubuntu=$(lsb_release -c -s)
-      if [ ! -r /etc/apt/sources.list.d/nflx-$ubuntu.list ]; then
-        echo "deb     [arch=amd64] http://repo.test.netflix.net:7001/artifactory/debian-local $ubuntu main" | sudo tee /etc/apt/sources.list.d/nflx-$ubuntu.list
-      fi
-      sudo apt-get update
-      sudo apt-get install -y --allow-unauthenticated libatlasclient g++ 
-      ;;
-  esac
-  mkdir -p build/Release
-  rm -f build/Release/libatlasclient.dylib
-  cp /usr/local/lib/libatlasclient.so build/Release/
-elif [ "$OS" = "Darwin" ]; then
-  brew tap | grep -q homebrew/nflx || brew tap homebrew/nflx https://stash.corp.netflix.com/scm/brew/nflx.git
-  if brew ls --versions nflx-atlas-client | grep -q "2\.2" ; then
-    echo nflx-atlas-client already installed: $(brew ls --versions nflx-atlas-client)
-  else
-    brew update
-    if brew ls --versions nflx-atlas-client >/dev/null 2>/dev/null; then
-      brew upgrade nflx-atlas-client || true
-    else
-      brew install nflx-atlas-client
-    fi
-  fi
-  mkdir -p build/Release
-  rm -f build/Release/libatlasclient.dylib
-  cp /usr/local/lib/libatlasclient.dylib build/Release/
+if [ -d nc/root/usr/local/include/atlas/atlas_client.h ]; then
+  echo Already installed
+  exit 0
 fi
+
+NATIVE_CLIENT_VERSION=${1:-v2.2.0}
```

Comment:
Consider defaulting to latest tag, something like:

```bash
git tag --sort committerdate | tail -n1
```

Reply:
I'm not sure about this. I think I'd like to move the version of the native-client used to `package.json` and make it very obvious so when people update the `version` of the node-package, then they see the version of the native-client being used. And if you want to change the version used, then you'll have to update the node package version as well.

Also note that I don't have the git repo at that point in the script. I'd have to use something like:

```
git ls-remote --tags https://github.com/Netflix-Skunkworks/atlas-native-client.git | ...
```

Reply:
I would just like to minimize the number of places we have to manually update to perform a release as well as make it consistent with other projects. With the java projects all we have to do is create a version tag and a travis build will be kicked off performing a release. Ideally the node client could be setup to work in the exact same way so we don't need to know a special procedure for this project.

Reply:
I was overlooking that this is the version for the transitive dep on the native client and not the version of this local project. So I agree package.json probably makes sense if we can make that the only place that needs to be updated.